skbio/embedding/_protein.py

```diff
@@ -7,10 +7,27 @@
 # ----------------------------------------------------------------------------
 from skbio.sequence import Protein
 from skbio.embedding._embedding import SequenceEmbedding
+from skbio.embedding._embedding import SequenceVector
+from skbio.stats.ordination import OrdinationResults
+from scipy.spatial.distance import pdist, squareform
 from skbio.util import get_data_path
 from pathlib import Path
 import pandas as pd
 import numpy as np
+from typing import List
+
+
+def _validate_protein(sequence):
+    if isinstance(sequence, Protein):
+        sequence = str(sequence)
+    elif isinstance(sequence, str):
+        if " " in sequence:
+            sequence = sequence.replace(" ", "")
+
+        # perform a check to make sure the sequence is a valid
+        # protein sequence
+        Protein(sequence)
```

Comment:
Will this print something to the screen? To avoid this behavior, maybe `_ = Protein(sequence)`?